packages/react-native-web/src/exports/Text/index.js

```diff
@@ -39,116 +39,118 @@ const forwardPropsList = {
 
 const pickProps = (props) => pick(props, forwardPropsList);
 
-const Text = forwardRef<TextProps, *>((props, forwardedRef) => {
-  const {
-    dir,
-    hrefAttrs,
-    numberOfLines,
-    onClick,
-    onLayout,
-    onPress,
-    onMoveShouldSetResponder,
-    onMoveShouldSetResponderCapture,
-    onResponderEnd,
-    onResponderGrant,
-    onResponderMove,
-    onResponderReject,
-    onResponderRelease,
-    onResponderStart,
-    onResponderTerminate,
-    onResponderTerminationRequest,
-    onScrollShouldSetResponder,
-    onScrollShouldSetResponderCapture,
-    onSelectionChangeShouldSetResponder,
-    onSelectionChangeShouldSetResponderCapture,
-    onStartShouldSetResponder,
-    onStartShouldSetResponderCapture,
-    selectable
-  } = props;
-
-  const hasTextAncestor = useContext(TextAncestorContext);
-  const hostRef = useRef(null);
-
-  const classList = [
-    classes.text,
-    hasTextAncestor === true && classes.textHasAncestor,
-    numberOfLines === 1 && classes.textOneLine,
-    numberOfLines != null && numberOfLines > 1 && classes.textMultiLine
-  ];
-  const style = [
-    props.style,
-    numberOfLines != null && numberOfLines > 1 && { WebkitLineClamp: numberOfLines },
-    selectable === true && styles.selectable,
-    selectable === false && styles.notSelectable,
-    onPress && styles.pressable
-  ];
-
-  useElementLayout(hostRef, onLayout);
-  useResponderEvents(hostRef, {
-    onMoveShouldSetResponder,
-    onMoveShouldSetResponderCapture,
-    onResponderEnd,
-    onResponderGrant,
-    onResponderMove,
-    onResponderReject,
-    onResponderRelease,
-    onResponderStart,
-    onResponderTerminate,
-    onResponderTerminationRequest,
-    onScrollShouldSetResponder,
-    onScrollShouldSetResponderCapture,
-    onSelectionChangeShouldSetResponder,
-    onSelectionChangeShouldSetResponderCapture,
-    onStartShouldSetResponder,
-    onStartShouldSetResponderCapture
-  });
-
-  function handleClick(e) {
-    if (onClick != null) {
-      onClick(e);
+const Text: React$AbstractComponent<TextProps, mixed> = forwardRef<TextProps, *>(
```

Comment:
`Text` should probably be typed more like `View`.

And rather than changing the indentation of all these files we should do something like RN https://github.com/facebook/react-native/blob/9be3356e02845a3a8194226a34faa4cb6f7295f1/Libraries/Components/ActivityIndicator/ActivityIndicator.js#L188-L192

Reply:
Fixed forwardRef. But leaving this open for indentation discussion as I just left those to lint